model_analyzer/config/run/run_config.py

```diff
@@ -75,6 +75,21 @@ def perf_config(self):
 
         return self._perf_config
 
+    def is_legal_combination(self):
+        """
+        Returns true if the run_config is valid and should be run. Else false
+        """
+        max_batch_size = 1
+        model_config = self._model_config.get_config()
+        if "max_batch_size" in model_config:
```

Comment:
Change this to be a one-liner:
max_batch_size = model_config['max_batch_size'] if "max_batch_size" in model_config else 1

Reply:
done